api.yml

```diff
@@ -19,9 +19,97 @@ paths:
+      requestBody:
+        content:
+          application/json:
+            schema:
+              description: The credential to be verified and a set of options to use when verifying the credential.
+              type: object
+              properties:
+                verifiableCredential:
+                  type: object
+                  additionalProperties: false
+                  description: The verifiable credential to check.
+                  example: {
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://www.w3.org/2018/credentials/examples/v1"
+  ],
+  "id": "http://example.edu/credentials/3732",
+  "type": [
+    "VerifiableCredential",
+    "UniversityDegreeCredential"
+  ],
+  "issuer": "did:example:c6f1c276e12ec21ebfeb1f712eb",
+  "issuanceDate": "2020-03-11",
+  "credentialSubject": {
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "degree": {
+      "type": "BachelorDegree",
+      "name": "Bachelor of Science and Arts"
+    }
+  },
+  "proof": [{
+    "type": "Ed25519Signature2018",
+    "created": "2020-03-11T23:09:06.803Z",
+    "jws": "eyJhbGciOiJFZERTQSIsImI2..hS5SPmWTFocjcDg",
+    "proofPurpose": "assertionMethod",
+    "verificationMethod": "did:example:c6f1c276e12ec21ebfeb1f712eb#jf893k"
+  }]
+}
+                options:
+                  type: object
+                  additionalProperties: false
+                  description: Options for specifying how the credential will be checked.
+                  properties:
+                    checks:
+                      type: array
+                      items:
+                        type: string
+                      description: An array of verification checks to be performed on the credential. WARNING - Implementers are cautioned that the list of checks is currently incomplete and other checks such as issuance, expiration, nonce, domain, and acceptable issuers are expected to be added in time. A default list is also expected to encapsulate all available checks. This option might be changed to a 'ignore' option where it lists the checks the developer would like to skip.
+                      example: ["proof"]
+      responses:
+        '200':
+          description: The verification was successful.
+          content:
+            application/json:
+              schema:
+                type: object
+                description: A description of the checks that were performed.
+                properties:
+                  checks:
+                    type: array
+                    items:
+                      type: string
+                    description: An array of verification checks that were performed. WARNING - Implementers are cautioned that the list of checks is currently incomplete and other checks such as issuance, expiration, nonce, domain, and acceptable issuers are expected to be added in time.
+                    example: ["proof"]
+        '400':
+          description: The verification failed.
```

Comment:
Based on my previous comment, I believe the intention is that in the case of multiple credentials, the API evaluates all credentials as a package -- so if one fails, the response will be 400. Suggested description: "one or more of the credential verifications failed."

Reply:
@mavarley when you leave a comment, you can click the icon with the plus button to suggest a change, and then the author can apply it by pressing a single button.
<img width="215" alt="Screen Shot 2020-03-12 at 12 27 22 PM" src="https://user-images.githubusercontent.com/8295856/76548513-d7e37b80-645c-11ea-9126-f2b3cc535e5a.png">

Reply:
```suggestion
          description: one or more of the credential verifications have failed.
```

Reply:
> I believe the intention is that in the case of multiple credentials
> The API evaluates all credentials as a package -- so if one fails, the response will be 400. Suggested description: "one or more of the credential verifications failed."

Hmm, you're talking about a presentation, which this API doesn't support (yet). You can't send multiple credentials to this endpoint, just one... we do intend to have an endpoint that verifies presentations, and in that case, it will only return 200 if *all* credentials and presentations were valid, and will return 400 if any credential or presentation failed. We're trying to focus this PR on credentials only and will handle your concern in a separate PR.

Issue #5 tracks this issue.